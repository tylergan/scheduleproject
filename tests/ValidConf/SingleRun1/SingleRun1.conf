on Tuesday at 1100 run /bin/echo hello there