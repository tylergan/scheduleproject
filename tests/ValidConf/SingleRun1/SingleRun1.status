will run at XX XX XX 11:00:00 2020 /bin/echo hello there
ran XX XX XX 11:00:00 2020 /bin/echo hello there
