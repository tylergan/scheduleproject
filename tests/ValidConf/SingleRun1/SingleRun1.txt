Testing that runner.py and runstatus.py outputs the correct output to STDOUT/STDERR when there is:
    1) A single command line in the config file