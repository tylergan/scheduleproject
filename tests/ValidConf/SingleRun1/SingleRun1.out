hello there
nothing left to run
